.travis.yml

```diff
@@ -1,14 +1,22 @@
 language: python
 python:
   - "2.7"
-branches:
-  only:
-  - /.*/
+matrix:
+  include:
+    - os: linux
+      dist: trusty
+    - os: osx
+      language: generic
 env:
   - SYSL_PLANTUML=http://www.plantuml.com/plantuml
+before_install:
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew list python &>/dev/null || brew install python; fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then export PATH=/usr/local/opt/python/libexec/bin:$PATH; fi
```

Comment:
You should not put a libexec directory on your path generally. .../python/bin should be sufficient. If it's not, it deserves a comment as to why.

Reply:
Brew suggests it as it's moving to a `python2` world:
```
> brew info python
==> Caveats
This formula installs a python2 executable to /usr/local/bin.
If you wish to have this formula's python executable in your PATH then add
the following to ~/.zshrc:
  export PATH="/usr/local/opt/python/libexec/bin:$PATH"

Pip and setuptools have been installed. To update them
  pip2 install --upgrade pip setuptools

You can install Python packages with
  pip2 install <package>

They will install into the site-package directory
  /usr/local/lib/python2.7/site-packages
```
https://github.com/Homebrew/homebrew-core/issues/15746

Reply:
I did not look closely enough in my /usr/local/opt/python/bin directory - it does NOT contain any executable called "python", only "python2", so .../python/bin is not a substitute for the libexec/bin directory. So carry on, and ignore me.